src/PackageArchive/Archive.CiServer.Patch.Compat/ArchiveBaseline.2.2.1.txt

```diff
@@ -0,0 +1,2 @@
+system.diagnostics.process\4.3.0\.signature.p7s
```

Comment:
I don't trust GitHub's diff viewer anymore. The file looks fine if I check it out locally.

![image](https://user-images.githubusercontent.com/2696087/51049816-d5993f00-1583-11e9-8ca2-b85e3c7dcc07.png)